envs/example/ci-full/group_vars/all.yml

```diff
@@ -1,17 +1,33 @@
 ---
-stack_env: example_ci-full
+stack_env: ci-full
+ursula_os: 'ubuntu'
```

Comment:
you are right. ursula_os is not needed in 31x. why I added it is that I copied whole ssl section in defaults-2.0.yml from master and it contained ursula_os.